### PR TITLE
fix kube controller command line args in 430 branch

### PR DESCRIPTION
**Pull request by yb01**
**What type of PR is this?**

> /kind bug

**What this PR does / why we need it**:
perf test cluster setup in 430 branch

### CODE REVIEW

File: cluster/gce/gci/configure-helper-common.sh [hunk 1]
```diff
@@ -2398,10 +2398,10 @@ function start-kube-controller-manager {
     params+=" --pv-recycler-pod-template-filepath-hostpath=$PV_RECYCLER_OVERRIDE_TEMPLATE"
   fi
   if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]]; then
-    RUN_CONTROLLERS="serviceaccount,serviceaccount-token,nodelifecycle"
+    RUN_CONTROLLERS="serviceaccount,serviceaccount-token,nodelifecycle,ttl"
```
Comment (Sindica) [inline cluster/gce/gci/configure-helper-common.sh:2401]: Add daemonset controller?

File: cluster/gce/gci/configure-helper-common.sh [hunk 2]
```diff
@@ -2398,10 +2398,10 @@ function start-kube-controller-manager {
...
   fi
   if [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
-    RUN_CONTROLLERS="*,-nodeipam,-nodelifecycle,-mizar-controllers,-network"
+    RUN_CONTROLLERS="*,-nodeipam,-nodelifecycle,-mizar-controllers,-network,-ttl"
```
Comment (Sindica) [inline cluster/gce/gci/configure-helper-common.sh:2404]: minus daemonset

File: cluster/gce/gci/configure-helper-common.sh (hunk 1 above)
Comment (yb01) [reply cluster/gce/gci/configure-helper-common.sh]: added

File: cluster/gce/gci/configure-helper-common.sh (hunk 2 above)
Comment (yb01) [reply cluster/gce/gci/configure-helper-common.sh]: removed

**Review by Sindica — APPROVED** [review]
/lgtm

**Pull request merged**